Content.Server/Storage/EntitySystems/EntityStorageSystem.cs

```diff
@@ -191,4 +199,33 @@ private void OnInsideExposed(EntityUid uid, InsideEntityStorageComponent compone
     }
 
     #endregion
+
+    #region Pry event handlers
+    private void OnGetPryMod(EntityUid uid, EntityStorageComponent component, ref GetPryTimeModifierEvent args)
+    {
+        args.BaseTime = component.PryTime;
+    }
+
+
+    private void OnBeforePry(EntityUid uid, EntityStorageComponent component, ref BeforePryEvent args)
+    {
+        // A simple crowbar won't be enough. You need a proper prying tool.
+        if (!args.PryPowered)
```

Comment:
Was unsure if `PryPowered` needed to be renamed as part of this, or if trying to re-use it (rather than creating a new field) is the preferred solution to begin with.

Left it as is now, but open to feedback.